concourse/pipelines/destroy-cloudfoundry.yml

```diff
@@ -136,41 +136,39 @@ jobs:
         tags: [colocated-with-web]
         file: paas-cf/concourse/tasks/get-cf-cli-config.yml
 
-      - task: remove-healthcheck-db
+      - task: unbind-and-delete-all-services
         tags: [colocated-with-web]
-        file: paas-cf/concourse/tasks/remove-db.yml
-        params:
-          ORG: admin
-          SPACE: healthchecks
-          DB_NAME: healthcheck-db
-          BOUND_APPS: healthcheck
+        config:
+          platform: linux
+          image_resource:
+            type: docker-image
+            source:
+              repository: ghcr.io/alphagov/paas/cf-cli
+              tag: 1b0fef4a74c771a23dbdf67cd830dc08b9c49258
+          inputs:
+            - name: paas-cf
+            - name: config
+          run:
+            path: sh
+            args:    
+                - -e
+                - -u
+                - -c
+                - |
+                  . ./config/config.sh
+                  if ! curl -I -f "${API_ENDPOINT}/info"; then
+                    echo "CF API unavailable. Skipping..."
+                    exit 0
+                  fi
 
-      - task: remove-billing-db
-        tags: [colocated-with-web]
-        file: paas-cf/concourse/tasks/remove-db.yml
-        params:
-          ORG: admin
-          SPACE: billing
-          DB_NAME: billing-db
-          BOUND_APPS: paas-billing-api paas-billing-collector
+                  cf api "${API_ENDPOINT}"
 
-      - task: remove-accounts-db
-        tags: [colocated-with-web]
-        file: paas-cf/concourse/tasks/remove-db.yml
-        params:
-          ORG: admin
-          SPACE: billing
-          DB_NAME: accounts-db
-          BOUND_APPS: paas-accounts
+                  if ! cf auth "${CF_ADMIN}" "${CF_PASS}"; then
+                    echo "Login failed.  Skipping..."
+                    exit 0
+                  fi
 
-      - task: remove-auditor-db
-        tags: [colocated-with-web]
-        file: paas-cf/concourse/tasks/remove-db.yml
-        params:
-          ORG: admin
-          SPACE: billing
-          DB_NAME: auditor-db
-          BOUND_APPS: paas-auditor
+                  ./paas-cf/scripts/unbind-and-delete-all-services.sh execute
```

Comment:
Script and pipeline contains no timeouts or retries. Do we want to consider this?